daemon/config/config_test.go

```diff
@@ -299,6 +300,28 @@ func TestValidateConfigurationErrors(t *testing.T) {
 				},
 			},
 		},
+		{
+			config: &Config{
```

Comment:
With quite a lot of test-cases now in this test, we might want to consider changing them to sub-tests, and adding a `name:` for each test.